cmake/Modules/FindFFMPEG.cmake

```diff
@@ -1,97 +1,18 @@
-# - Try to find ffmpeg libraries (libavcodec, libavformat and libavutil)
+# - Try to find ffmpeg executable
 # Once done this will define
 #
-# FFMPEG_FOUND - system has ffmpeg or libav
-# FFMPEG_INCLUDE_DIR - the ffmpeg include directory
-# FFMPEG_LIBRARIES - Link these to use ffmpeg
-# FFMPEG_LIBAVCODEC
-# FFMPEG_LIBAVFORMAT
-# FFMPEG_LIBAVUTIL
-#
-# Copyright (c) 2008 Andreas Schneider <mail@cynapses.org>
-# Modified for other libraries by Lasse Kärkkäinen <tronic>
-# Modified for Hedgewars by Stepik777
-#
-# Redistribution and use is allowed according to the terms of the New
-# BSD license.
-#
-if (FFMPEG_LIBRARIES AND FFMPEG_INCLUDE_DIR)
```

Comment:
Removing that is gonna break the headless recorder, as it does use FFMPEG.
How about leaving this file as is, and adding current changes in a new file `FindFFMPEGEXEC.cmake` ?